internal/core/src/segcore/ChunkedSegmentSealedImpl.cpp

```diff
@@ -2599,7 +2599,7 @@ ChunkedSegmentSealedImpl::RemoveFieldFile(const FieldId field_id) {
 
 void
 ChunkedSegmentSealedImpl::LazyCheckSchema(SchemaPtr sch) {
-    if (sch->get_schema_version() > schema_->get_schema_version()) {
```

Comment:
for fields needed doing physical backfill, fill with null_values is meaningless
we will make search get around for segments lack of physical_backfill_fields